sklearn/pipeline.py

```diff
@@ -673,13 +683,13 @@ def _validate_transformers(self):
                                 "transform. '%s' (type %s) doesn't" %
                                 (t, type(t)))
 
-    def _iter(self):
+    def _iter(self, X=None, skip_none=True):
         """Generate (name, est, weight) tuples excluding None transformers
```

Comment:
skip_none in particular needs to be documented because the need for it was quite subtle and will be forgotten